docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -409,138 +395,111 @@ EOF
 terraform apply
 ----
 
-. Create the first shown DNS records
-
-. Wait for the DNS records to propagate!
-+
-[source,console]
-----
-sleep 600
-host "api.${CLUSTER_ID}.${BASE_DOMAIN}"
-----
+. Set up DNS NS records on parent zone using the data from the Terraform output variable `dns_entries` from the previous step
 
-. Provision master nodes
+. Create LB hieradata
 +
-[source,console]
+[source,bash]
 ----
 cat > override.tf <<EOF
 module "cluster" {
-  bootstrap_count = 1
+  bootstrap_count = 0
+  master_count    = 0
   infra_count     = 0
+  storage_count   = 0
   worker_count    = 0
 }
 EOF
-
-terraform apply
+terraform apply -target "module.cluster.local_file.lb_hieradata[0]"
 ----
 
-. Add the remaining shown DNS records to the previous ones.
+. Review and merge the LB hieradata MR (listed in Terraform output `hieradata_mr`) and wait until the deploy pipeline after the merge is completed.
 
-. Wait for bootstrap to complete
+. Create LBs
 +
-[source,console]
+[source,bash]
 ----
-openshift-install --dir ../../../target \
-  wait-for bootstrap-complete
+terraform apply
 ----
 
-. Remove bootstrap node and provision infra nodes
+
+////
+########################
+TODO fix command below
+########################
+////
```

Comment:
What's the expected timeline for fixing the command? Should be done soon. Probably something like my suggestion below would work for now.

Reply:
I'd hoped to come up with a fix in this PR